pkg/deploy-manager/storagecluster.go

```diff
@@ -350,13 +350,22 @@ func (t *DeployManager) labelWorkerNodes() error {
 	}
 
 	labeledCount := 0
+	arbiterZone, err := t.GetArbiterZone()
```

Comment:
This is a node label, right? This may help clarify.
```suggestion
	arbiterZoneLabel, err := t.GetArbiterZoneLabel()
```